docker/r10k/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:16.04
+
+ARG vcs_ref
+ARG build_date
+ARG version="2.5.5"
```

Comment:
Does this just set a default if one isn't passed in?

Reply:
yeah, but I should update this to at least something in the 2.6 series probably, will update that